stdlib/2and3/bz2.pyi

```diff
@@ -1,6 +1,34 @@
 # Stubs for bz2
 
-# TODO: This stub is incomplete
+from typing import Any, BinaryIO, TextIO, IO, Union
 
 def compress(data: bytes, compresslevel: int = ...) -> bytes: ...
 def decompress(data: bytes) -> bytes: ...
+
+def open(filename: Union[str, bytes, IO[Any]],
+         mode: str = 'rb',
+         encoding: str = None,
+         errors: str = None,
+         newline: str = None) -> Union[TextIO, BinaryIO]: ...
```

Comment:
If the return type depends on the mode string, the convention is to return `IO[Any]`, as otherwise the return type may require `isinstance` tests to use, which we generally don't want.